APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -1141,6 +1208,29 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
                 }
             }
                 break;
+                
+            case kAPCUserInfoItemTypeDateOfBirth:
+            {
+                if (self.canEditBirthDate &&
+                    ![self isEditingAllowedForHealthKitProperty:HKCharacteristicTypeIdentifierDateOfBirth])
+                {
+                    [self showEditingNotAllowedAlertForHealthKitProperty:HKCharacteristicTypeIdentifierDateOfBirth];
+                    [tableView deselectRowAtIndexPath:indexPath animated:YES];
+                }
+                else
+                {
+                    APCTableViewItem *field = [self itemForIndexPath:indexPath];
+                    if (self.isEditing && field.isEditable) {
+                        [super tableView:tableView didSelectRowAtIndexPath:indexPath];
+                    }
+                }
+            }
+                break;
+                
+            case kAPCUserInfoItemTypeHeight:
+            {
+                [self setEditing:!self.isEditing];
```

Comment:
As a recommendation, add a line before any debugging code:

#warning DO NOT COMMIT

Sent from my iPhone

> On Jun 2, 2016, at 7:16 AM, Michael DePhillips <notifications@github.com> wrote:
> 
> In APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m:
> 
> > +                    [self showEditingNotAllowedAlertForHealthKitProperty:HKCharacteristicTypeIdentifierDateOfBirth];
> > +                    [tableView deselectRowAtIndexPath:indexPath animated:YES];
> > +                }
> > +                else
> > +                {
> > +                    APCTableViewItem *field = [self itemForIndexPath:indexPath];
> > +                    if (self.isEditing && field.isEditable) {
> > +                        [super tableView:tableView didSelectRowAtIndexPath:indexPath];
> > +                    }
> > +                }
> > +            }
> > +                break;
> > +                
> > +            case kAPCUserInfoItemTypeHeight:
> > +            {
> > +                [self setEditing:!self.isEditing];
> Yes it would. It was debug code I left in, glad you spotted it!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>